Solutions/Lucas/ex_1/gunicorn_starter.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+gunicorn -w 4 -b 0.0.0.0:5000 app:app
```

Comment:
Para imprimir o log no terminal do container poderia utilizando o argumento : `--access-logfile=-` , desta forma torna o processo de debugging mais fácil.
ref: https://docs.gunicorn.org/en/stable/settings.html#logging

```suggestion
gunicorn -w 4 -b 0.0.0.0:5000 app:app --access-logfile=-
```